contracts/verifier/test/VerifierBaseTester.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.13;
+
+import { EIP712 } from "@openzeppelin/contracts/utils/cryptography/EIP712.sol";
+import { VerifierBase } from "../VerifierBase.sol";
+
+/// @dev Empty implementation for the sole purpose of testing base class
+contract VerifierBaseTester is VerifierBase {
+    constructor() EIP712("Perennial Root Unit Tests", "1.0.0") { }
```

Comment:
nit: this library is not actually a Perennial specific thing -- we use it across a few different projects.

Reply:
resolved in c8a7201b2c82bcba237def531028fd902c4c99eb